Sources/ComposableArchitecture/Store.swift

```diff
@@ -91,7 +91,7 @@ public final class Store<State, Action> {
   ///   - fromLocalAction: A function that transforms `LocalAction` into `Action`.
   /// - Returns: A publisher of stores with its domain (state and action) transformed.
   public func scope<P: Publisher, LocalState, LocalAction>(
-    state toLocalState: @escaping (AnyPublisher<State, Never>) -> P,
+    statePublisher toLocalState: @escaping (AnyPublisher<State, Never>) -> P,
```

Comment:
one other option to consider is renaming the function since it returns a publisher:

```swift
func publisherScope(...)
func scopePublisher(...)
```

Reply:
and should we do an `unavailable` release to help people out before fully removing?

Reply:
Ah yeah I actually named it that first that but ended up changing to the argument label is since actions aren't scoped by publisher:

`scope(statePublisher:action:)` vs. `publisherScope(state:action:)`

Thoughts?

Reply:
I think `publisherScope` since it describes what's be returned, and it's nice to keep the shortness of `state:` and `action`:.

Reply:
Done! Along with deprecations. I think it reads nicely.